### DISCUSSION TITLE
exclude column form selection (Hidden Columns) `@select(false)`

### DISCUSSION THREAD

**Discussion by satishbabariya** [prisma/prisma#2159] [Q&A] [answered]
How can i exclude certain column form select query overall like Password in the User Model.

Currently i'm using `@Column({ select: false  })` in typeorm. 
ref: https://github.com/satishbabariya/nodejs-boilerplate/blob/master/src/models/User.ts#L37

Is ther any way to do this in prisma2? like

```prisma
model User {
  id       Int    @default(autoincrement()) @id
  email    String @unique
  password String @select(false)
}
```

reference of TypeORM: https://typeorm.io/#/select-query-builder/hidden-columns

**Comment by ryands17** [comment, marked as answer]
Feature Request referenced [here](https://github.com/prisma/prisma/issues/2170)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
